webpack.config.js

```diff
@@ -1,36 +1,40 @@
+const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ImagesConfigWebpackPlugin = require('image-config-webpack-plugin');
 const JsConfigWebpackPlugin = require('js-config-webpack-plugin');
 const ScssConfigWebpackPlugin = require('scss-config-webpack-plugin');
-const path = require('path');
 
 module.exports = {
-  entry: ['@babel/polyfill', './src/index.js'],
-  resolve: {
```

Comment:
this has not to be deleted too, it tells webpack where to find `@babel/polyfill`

Reply:
cool @T2Wil , I didn't know when I have deleted it